spirekey/src/components/Registration/Registration.tsx

```diff
@@ -62,6 +63,16 @@ export default function Registration({
     router.push(`${completeRedirectUrl}?${new URLSearchParams({ user })}`);
   };
 
+  const handleRecoverAccountHandler = async () => {
```

Comment:
We should call the on complete here like the on register

Reply:
done..that fixes the SDK